### PR TITLE
Update ledvance.ts

### PR DESCRIPTION
**Pull request by Koenkk**

### CODE REVIEW

**Review by dhanukapulkit — COMMENTED** [review]
Resolved errors and new plugs added

**Review by dhanukapulkit — COMMENTED** [review]
Libraries included

File: src/devices/ledvance.ts [hunk 1]
```diff
@@ -38,6 +44,38 @@ const definitions: Definition[] = [
         description: 'Panel TW LED 625 UGR19',
         extend: [ledvanceLight({colorTemp: {range: [153, 370]}, color: true})],
     },
+    {
+        zigbeeModel: ['PLUG EU EM T'],
+        model: '4099854295232',
+        vendor: 'LEDVANCE',
+        description: 'SMART+ Indoor Plug EU with Energy meter ',
+        fromZigbee: [fz.on_off, fz.metering],
```
Comment (Koenkk) [inline src/devices/ledvance.ts]: Please remove, configure fromZigbee, toZigbee and use: `extend: [onOff(), electricityMeter()],`

**Review by dhanukapulkit — COMMENTED** [review]
Added extend: [onOff(), electricityMeter()],

**Pull request merged**